test/algebra/dawg-syntax/bnodes/syntax-bnodes-01.json

```diff
@@ -6,7 +6,7 @@
       {
         "subject": {
           "termType": "BlankNode",
-          "value": "b0"
+          "value": "e_b0"
```

Comment:
I think the changes here (and in all the other test files) aren't necessary anymore due to the canonicalize function?

Reply:
Most likely so. Well spotted! I'll give this a look.